### ISSUE TITLE
_scalar_type change for Windows 7 GPU runnig

### ISSUE THREAD

**Issue by wyasuda** [pfnet/chainer#423]
I had installed chainer to my Windows7 PC and could run the command below.

> python chainer/examples/mnist/train_mnist.py

Then I tried to run the command below.

> python chainer/examples/mnist/train_mnist.py --gpu=0

I got an error message below.

```
Traceback (most recent call last):
  File "chainer/examples/mnist/train_mnist.py", line 81, in <module>
    loss, acc = forward(x_batch, y_batch)
  File "chainer/examples/mnist/train_mnist.py", line 61, in forward
    return F.softmax_cross_entropy(y, t), F.accuracy(y, t)
  File "C:\Python27\lib\site-packages\chainer\functions\loss\softmax_cross_entropy.py", line 131, in softmax_cross_entropy
    return SoftmaxCrossEntropy(use_cudnn, normalize)(x, t)
  File "C:\Python27\lib\site-packages\chainer\function.py", line 174, in __call__
    outputs = self.forward(in_data)
  File "C:\Python27\lib\site-packages\chainer\function.py", line 236, in forward
    return self.forward_gpu(inputs)
  File "C:\Python27\lib\site-packages\chainer\functions\loss\softmax_cross_entropy.py", line 60, in forward_gpu
    )(t, y.reduced_view(), y.shape[-1], -1.0 / count)
  File "C:\Python27\lib\site-packages\cupy\reduction.py", line 392, in __call__
    _check_args(in_args + out_args)
  File "C:\Python27\lib\site-packages\cupy\elementwise.py", line 76, in _check_args
    raise TypeError('Unsupported type %s' % type(arg))
TypeError: Unsupported type <type 'long'>
```

Therefore, I edited line 56 of File "C:\Python27\lib\site-packages\cupy\elementwise.py".

```
_scalar_type = (int, long, float, bool) + tuple(t.type for t in _typenames.keys())
```

I just added type 'long' to _scalar_type.
Then I could succeeded to run the command above.

Are there anybody who would comment on this?
Am I doing right thing?
Thanks in advance.

**Comment by okuta** [comment]
Thank you for your comment.

This problem was fixed in v1.3.1.
https://github.com/pfnet/chainer/blob/v1.3.1/cupy/elementwise.py#L56

Pleese try latest version.

**Comment by wyasuda** [comment]
Thanks!
I upgraded chainer to v1.3.1 and confirmed that it run properly as you mentioned.

By the way, I use NEXTGEAR-NOTE i5700GA2 and GPU is 50 times faster than CPU!
Just for your information.
